libs/blocks/merch-card/merch-card.js

```diff
@@ -1,184 +1,186 @@
 /* eslint-disable prefer-destructuring */
 import { decorateButtons, decorateBlockHrs } from '../../utils/decorate.js';
 import { loadStyle, getConfig, createTag } from '../../utils/utils.js';
-import { addBackgroundImg, addWrapper, addFooter } from '../card/cardUtils.js';
+ import { getUpFromSectionMetadata } from '../card/cardUtils.js';
+import { decorateLinkAnalytics } from '../../martech/attributes.js';
 import { replaceKey } from '../../features/placeholders.js';
+import '../../deps/commerce.js';
+import '../../deps/merch-card.js';
+import { initLegacyMerchCard } from './legacy-merch-card.js';
```

Comment:
let's import legacy-merch-card only if needed. Otherwise it will impact performance.

Reply:
Fixed